content/intro-to-storybook/vue/en/deploy.md

```diff
@@ -1,99 +1,155 @@
 
 ## Exporting as a static app
 
-To deploy Storybook we first need to export it as a static web app. This functionality is already built into Storybook, we just need to change it like we did before when the project was initialized in the [get started section](/vue/en/get-started).
-
-```javascript
-{
-  "scripts": {
-   "build-storybook": "build-storybook -s public"
-  }
-}
-```
+To deploy Storybook we first need to export it as a static web app. This functionality is already built-in to Storybook and pre-configured.
 
-Now when you run Storybook via `yarn build-storybook`, it will output a static Storybook in the `storybook-static` directory.
+Running `yarn build-storybook` will output a static Storybook in the `storybook-static` directory, which can then be deployed to any static site hosting service.
 
-## Continuous deploy
+## Publish Storybook
 
-We want to share the latest version of components whenever we push code. To do this we need to continuous deploy Storybook. We’ll rely on GitHub and Netlify to deploy our static site. We’re using the Netlify free plan.
+This tutorial uses <a href="https://www.chromatic.com/">Chromatic</a>, a free publishing service made by the Storybook maintainers. It allows us to deploy and host our Storybook safely and securely in the cloud.
 
-### GitHub
+### Setup a repository in GitHub
 
-If you're following along from the previous testing chapter jump to setting up a repository on GitHub.
+Before we begin, our local code needs to sync with a remote version control service. When our project was initialized in the [Get started chapter](/intro-to-storybook/vue/en/get-started), Vue CLI already created a local repository for us. At this stage it's safe to add our files to the first commit.
 
-When the project was initialized with Vue CLI, a local repository was already setup for you. At this stage it's safe to add the files to the first commit.
+Issue the following commands to add and commit the changes we've done so far.
 
 ```bash
 $ git add .
 ```
 
-Now commit the files.
+Followed by:
 
 ```bash
 $ git commit -m "taskbox UI"
 ```
 
-### Setup a repository in GitHub
-
-Go to GitHub and setup a repository [here](https://github.com/new). Name your repo “taskbox”.
+Go to GitHub and create a new repository for our project [here](https://github.com/new). Name the repo “taskbox”, same as our local project.
 
 ![GitHub setup](/intro-to-storybook/github-create-taskbox.png)
 
-In the new repo setup copy the origin URL of the repo and add it to your git project with this command:
+In the new repo, grab the origin URL of the repo and add it to your git project with this command:
 
 ```bash
 $ git remote add origin https://github.com/<your username>/taskbox.git
 ```
 
-Finally push the repo to GitHub
+Finally, push our local repo to the remote repo on GitHub with:
 
 ```bash
 $ git push -u origin master
 ```
```

Comment:
@andre-brdoch replace master with main. GitHub recently updated the branch naming policy and with that the tutorials were updated as well to reflect this